deployment/docker/alerts-health.yml

```diff
@@ -58,6 +58,18 @@ groups:
             Too high CPU usage may be a sign of insufficient resources and make process unstable.
             Consider to either increase available CPU resources or decrease the load on the process.
 
+      - alert: TooHighGoroutineSchedulingLatency
+        expr: histogram_quantile(0.99, sum(rate(go_sched_latencies_seconds_bucket{job="vmagent-monitoring-vmagent"}[5m])) by (le, job, instance)) > 0.1
```

Comment:
I'm not sure the `job="vmagent-monitoring-vmagent"` filter is needed here. @hagen1778 , @f41gh7

Reply:
I think, we have to add some filter to it (maybe some generic ~"vmagent|vmalert"? Because metric with the same name produced by `prometheus_client` lib. And it could produce alerts not related to the VM.

 It'd be great to have some kind of templating for such alerts.

Reply:
Sorry, it is a typo https://github.com/VictoriaMetrics/VictoriaMetrics/commit/a5c002edef47e9165c9dac008053907b5fa2133a